docs/inference.rst

```diff
@@ -354,17 +355,17 @@ Now run::
     # PSD estimation options
     PSD_ESTIMATION="H1:median L1:median"
     PSD_INVLEN=4
```

Comment:
No, you can't set the inverse specturm length to be 16 here. If you did, you'd have to make the gps start/end times be 64s because the beginning/end 16s would be corrupted. That long of a segment would make the PE job take forever. Four seconds seems to be long enough. For this reason, we can't make the PSD be exactly the same as is used in the search, but better to be close.